src/core/build_label_test.go

```diff
@@ -94,6 +94,16 @@ func TestSubrepoLabel(t *testing.T) {
 	assert.EqualValues(t, BuildLabel{PackageName: "", Name: ""}, label.SubrepoLabel())
 }
 
+func TestParseBuildLabelParts(t *testing.T) {
+	target1 := "@unittest_cpp//:unittest_cpp"
+	targetNewSyntax := "@unittest_cpp"
+	pkg, name, subrepo := parseBuildLabelParts(target1, "/", nil)
+	pkg2, name2, subrepo2 := parseBuildLabelParts(targetNewSyntax, "/", nil)
+	assert.Equal(t, pkg, pkg2)
```

Comment:
probably better to assert the values directly; technically you could pass this test with an implementation that always returned "" for the subrepo for example.